### ISSUE TITLE
NTR: Digits: increased number of flexion creases (Extremities, joints & nails)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7966]
New term request Digits: increased number of flexion creases (elthat_110106040716): Extra

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000109
- label: Supernumerary finger flexion crease
- parents: [Abnormal finger flexion crease (HP:0006143)](https://hpo.jax.org/app/browse/term/HP:0006143)
- pmid: PMID:12244551
- def: The presence of an extra digital crease in and individual with normal joint anatomy.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7966
- synonyms:
